web/src/features/fireWeather/components/graphs/PrecipGraph.tsx

```diff
@@ -0,0 +1,364 @@
+import React, { useRef, useEffect, useMemo } from 'react'
+import * as d3 from 'd3'
+import { makeStyles } from '@material-ui/core/styles'
+import moment from 'moment'
+
+import { ObservedValue } from 'api/observationAPI'
+import * as d3Utils from 'utils/d3'
+import { formatDateInPDT } from 'utils/date'
+import { NoonForecastValue } from 'api/forecastAPI'
+import { ToggleValues } from 'features/fireWeather/components/graphs/useGraphToggles'
+import { PDT_UTC_OFFSET } from 'utils/constants'
+
+const observedPrecipColor = '#a50b41'
+const forecastPrecipColor = '#fb0058'
+
+const useStyles = makeStyles({
+  root: {
+    paddingBottom: 15,
+
+    '& .yAxisLabel': {
+      textAnchor: 'middle',
+      font: '9px sans-serif'
+    },
+
+    '& .tooltip': {
+      pointerEvents: 'none',
+      font: '8.5px sans-serif',
+
+      '&__cursor': {
+        strokeWidth: 1,
+        stroke: 'gray',
+        strokeDasharray: '1,1',
+        opacity: 0
+      },
+
+      '&--hidden': {
+        display: 'none'
+      }
+    },
+
+    '& .precipLine': {
+      '&__observed': {
+        strokeWidth: 2.5,
+        stroke: observedPrecipColor
+      },
+
+      '&__forecast': {
+        strokeWidth: 2.5,
+        stroke: forecastPrecipColor
+      },
+
+      '&--hidden': {
+        visibility: 'hidden'
+      }
+    }
+  }
+})
+
+interface PrecipValue {
+  date: Date
+  precip?: number
```

Comment:
could you please rename this to `observedPrecip` (assuming that's what it is?) I think it would be better to make it explicit, especially once we start adding to this with the different model types, etc.

Reply:
Will do